onex/chain.json

```diff
@@ -0,0 +1,123 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "onex",
+  "status": "live",
+  "website": "https://app.onomy.io/",
+  "network_type": "mainnet",
+  "pretty_name": "ONEX",
+  "chain_id": "onex-mainnet-1",
+  "bech32_prefix": "onomy",
```

Comment:
This bech32 prefix is already taken by Onomy

Reply:
the bech32 prefix is set to onomy. community voted to keep prefixes the same for the onomy ecosystem so that all tokens in the ecosystem can be sent to the same address. ONEX is a consumer chain to the Onomy parent chain.